AnnoyancesFilter/sections/popups.txt

```diff
@@ -96,6 +96,7 @@ atlanticleague.com##.incap_page-tooltip
 58pic.com##.activitySettings-fixedBottom
 global.llbean.com##.subscribe-modal
 global.llbean.com##.ui-widget-overlay
+nymag.com##.modal.right-aligned
```

Comment:
As I understand this rule hides newsletter popup, if so, it looks okay, but maybe it would be better to use:
`nymag.com##.newsletter-slideout`